Lib.Web.Mvc/JQuery/JqGrid/DataAnnotations/JqGridColumnSearchableAttribute.cs

```diff
@@ -176,5 +226,24 @@ protected override void InternalOnMetadataCreated(ModelMetadata metadata)
             metadata.SetColumnSearchType(SearchType);
         }
         #endregion
+
+        #region Overrides of JqGridColumnElementAttribute
+
+        /// <summary>When overriden in delivered class, provides a list of events to apply to the element.</summary>
+        protected override IList<JqGridColumnDataEvent> DataEvents
```

Comment:
Exactly the same thing as with HtmlAttributes property - it is expected by end users to override this without breaking anything.